src/Calculations.ts

```diff
@@ -8,11 +8,10 @@ export function computeAngle(points: Array<Point3D|null>): number|null {
     if (points.length < 3)
         return null;
 
-    const first3HasNullOrNaN = points.slice(0,3)
-        .some(point => point===null||isNaN(point.x)||isNaN(point.y)||isNaN(point.z));
+    const first3HasNull = points.slice(0,3).some(point => point===null);
 
-    if (first3HasNullOrNaN)
-        return null;
+    if (first3HasNull) //null markers are hidden in RenderView (and aren't selectable),
+        return null;   //so this case shouldn't be true except for in testing
```

Comment:
not quite true, a selected point might become null when moving through time

Reply:
True. I forgot the user might take an angle measurement while paused, leave the markers selected, and drag the timeline thumb to another moment. I was picturing taking measurements while the scene is static. Fixed